### PR TITLE
MailHandler: Use array access to compare first character

### PR DESCRIPTION
**Pull request by glensc**

### CODE REVIEW

File: src/Monolog/Handler/MailHandler.php [hunk 1]
```diff
@@ -70,7 +70,7 @@ protected function getHighestRecord(array $records): array
 
     protected function isHtmlBody(string $body): bool
     {
-        return substr($body, 0, 1) === '<';
+        return $body[0] === '<';
```
Comment (xabbuh) [inline src/Monolog/Handler/MailHandler.php:73]: This will break when an empty string is passed.

Comment (glensc) [reply src/Monolog/Handler/MailHandler.php:73]: @xabbuh `$body` is always `string` (type hinted), so that can not happen!

**Comment by Seldaek** [comment]
Sorry if I'm stupid here but.. what's the point of this change exactly?

**Comment by glensc** [comment]
@Seldaek my ide suggested to use array access rather function call. typically it's considered better practice to use array offset access for single byte rather substr call.

**Comment by Seldaek** [comment]
Ok sure why not I suppose..

**Pull request merged**